src/main/java/duke/command/ShowCommand.java

```diff
@@ -0,0 +1,59 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.CalendarException;
+
+public class ShowCommand extends Command {
```

Comment:
Should the class name be a noun instead? I noticed the same issue with the naming of several other classes as well.